cg/store/crud/read.py

```diff
@@ -1736,11 +1737,16 @@ def get_all_pools_to_deliver(self) -> list[Pool]:
         )
         return records.all()
 
-    def get_orders(self, limit: int | None = None) -> list[Order]:
+    def get_orders(self, orders_request: OrdersRequest | None = None) -> list[Order]:
```

Comment:
`orders_request` cannot be none here, right?

Reply:
Fixed